pkg/util/kvgraph/util.go

```diff
@@ -72,9 +72,7 @@ func addVolumeGraph(vmiSpec v1.VirtualMachineInstanceSpec, vmName, namespace str
 		switch {
 		case volume.DataVolume != nil:
 			resources = addVeleroResource(volume.DataVolume.Name, namespace, "datavolumes", resources)
-			if isBackup {
```

Comment:
The `isBackup` variable isn't needed anymore in this or even the calling functions. We should clean it up.

Reply:
right.missed that from previous PR